staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff.go

```diff
@@ -747,13 +747,15 @@ func (az *Cloud) processHTTPRetryResponse(service *v1.Service, reason string, re
 	}
 
 	if shouldRetryHTTPRequest(resp, err) {
+		message := "processHTTPRetryResponse: backoff failure, will retry"
+		if resp != nil {
+			message = fmt.Sprintf("%s, HTTP response: %d", message, resp.StatusCode)
+		}
 		if err != nil {
```

Comment:
if `resp != nil` && `err != nil`, then we only get second error msg, is that what you want? @feiskyer

Reply:
it would include both